db/srm/reflection.go

```diff
@@ -218,6 +218,15 @@ func FieldRecipientsFromType(logger logging.Logger, sqlFields []string,
 	return FieldRecipientsFromValueOf(logger, sqlFields, fieldMap, vod)
 }
 
+// noopScanner implements the Scanner interface and ignores the value
+// this is useful if we do not know what to do with a value.
+// ie. We have asked for "*" and receive some unmapped fields
+type noopScanner struct{}
```

Comment:
Two comments:
1) have `noopScanner` receive logger so either the scan logs that a value/field is being dropped 
2) Add your name/data to CONTRIBUTORS you are well over 10 lines.